Say Their Names.xcodeproj/project.pbxproj

```diff
@@ -1203,6 +1207,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = H4TYCSEM4B;
```

Comment:
please revert these 🙏

Reply:
Sorry about these. I forgot to clear them up before submitting. WIll remove those.

Reply:
No worries! it's a common thing, happens to many here :)